### PR TITLE
public: remove extra whitespace from metainfo

### PR DESCRIPTION
**Pull request by jkozol**

### CODE REVIEW

**Review by ondrejbudai — COMMENTED** [review]
Looks good to my untrained eyes. :)

File: cockpit-composer.spec.in [hunk 1]
```diff
@@ -31,6 +31,7 @@ the cloud. It integrates into Cockpit as a frontend for osbuild.
 mkdir -p %{buildroot}/%{_datadir}/cockpit/composer
 cp -a public/* %{buildroot}/%{_datadir}/cockpit/composer
 mkdir -p %{buildroot}/%{_datadir}/metainfo/
+appstream-util validate-relax --nonet public/io.weldr.cockpit-composer.metainfo.xml
```
Comment (martinpitt) [inline cockpit-composer.spec.in:34]: You need a new `BuildRequires: libappstream-glib` for that. It works in our CI as that is pre-installed into the mocks, but it'll fail in koji/brew.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks!

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**